local-requirements.txt

```diff
@@ -38,3 +38,4 @@ Markdown==3.2.2
 setuptools~=54.2.0
 whitenoise==5.2.0
 coverage
+rocrate==0.6.1
```

Comment:
I suggest switching to the latest version (0.7.0). Among other things, it fixes a performance bug.

Reply:
@simleo many thanks for taking the time to check this

Reply:
updated to 0.7.0 in https://github.com/FAIRDataPipeline/data-registry/commit/4e5ad7db1d2f5314592a405f18f23491dbd11d65